d2bs/kolbot/libs/AutoMule.js

```diff
@@ -414,16 +427,52 @@ MainLoop:
 		return true;
 	},
 
+	matchItem: function (item, list) {
+		var i, info, parsedLine,
+			parsedPickit = [], classIDs = [];
+
+		for (i = 0; i < list.length; i += 1) {
+			info = {
+				file: "Character Config",
+				line: list[i]
+			};
+
+			if (typeof list[i] === "number") { // classids
+				classIDs.push(list[i]);
+			} else if (typeof list[i] === "string") { // pickit entries
+				parsedLine = NTIP.ParseLineInt(list[i], info);
+
+				if (parsedLine) {
+					parsedPickit.push(NTIP.ParseLineInt(list[i], info));
```

Comment:
should be `.push(parsedLine)`?

Reply:
It did seem redundant but this was the way it was done by Kolton in Runewords.js so I thought there was reason behind this. Maybe there wasn't.